plugins/inputs/statsd/statsd.go

```diff
@@ -489,6 +520,9 @@ func (s *Statsd) parseStatsdLine(line string) error {
 
 	lineTags := make(map[string]string)
 	if s.ParseDataDogTags {
+		log.Printf("I! WARNING statsd: parse_data_dog_tags config option is deprecated,  please use datadog_extensions instead")
```

Comment:
Something like this, with W! and the [inputs.statsd] bit.  Also would be nice to wrap this in the code near 80 chars.
```
log.Println("W! [inputs.statsd] The parse_data_dog_tags option has been deprecated, please use datadog_extensions instead.")
```

Move this up to line 325, don't use the s.ParseDataDogTags field any other places in the code.